app/poc-common/form-display/directives/formField.js

```diff
@@ -25,6 +25,28 @@ angular.module('poc.common.formdisplay')
             
         var formLogic = {};
         
+        var fireHideEvent = function (event) {
+             $scope.$watch('aForm.' + $scope.fieldId + '.$viewValue', function (newVal, oldVal) {
+                     if (newVal !== oldVal && !_.isUndefined(oldVal)) {
+                         $rootScope.$broadcast(event, newVal);
+                     }
+                 });
+         };
+
+         var listenHideEvent = function (event) {
+             $scope.$on(event, function (event, val) {
+                     var valJson = JSON.parse(val);
+                     if (valJson.display === "YES") {
```

Comment:
Thank you for the observation. Would it be a good idea if i change the **display** value for the **uuid** of the answer?